### PR TITLE
feat: add new option session_forced_logout_timeout

### PR DESCRIPTION
**Pull request by voroyam**
add `OWNCLOUD_SESSION_FORCED_LOGOUT_TIMEOUT` variable.

It expires the session and logs the user out after set time.

### CODE REVIEW

File: v20.04/overlay/etc/templates/config.php [hunk 1]
```diff
@@ -90,11 +90,15 @@ function getConfigFromEnv() {
   if (getenv('OWNCLOUD_SESSION_LIFETIME') != '') {
     $config['session_lifetime'] = (int) getenv('OWNCLOUD_SESSION_LIFETIME');
   }
-
+
```
Comment (xoxys) [inline v20.04/overlay/etc/templates/config.php:93]: Fix whitespacing.

**Review by xoxys — CHANGES_REQUESTED** [review]
See comments.

**Comment by voroyam** [comment]
> * `latest` image missing

I don't understand what you mean

File: README.md [hunk 2]
```diff
@@ -321,7 +321,7 @@ ownCloud Docker base image.
 - `OWNCLOUD_SESSION_LIFETIME=` \
   Define the lifetime of a session after inactivity (see [documentation](https://doc.owncloud.com/server/latest/admin_manual/configuration/server/config_sample_php_parameters.html#define-the-lifetime-of-a-session-after-inactivity)).
 - `OWNCLOUD_SESSION_FORCED_LOGOUT_TIMEOUT=` \
-  Expires the session and logs the user out after set time.
+  Expires the session and logs the user out after set time. (see [documentation](https://doc.owncloud.com/server/next/admin_manual/configuration/server/config_sample_php_parameters.html#enable-to-force-user-logout)).
```
Comment (xoxys) [inline README.md:324]: Please use `/latest/` links not `/next/`

**Review by xoxys — CHANGES_REQUESTED** [review]

**Comment by xoxys** [comment]
> I don't understand what you mean

There are two images, one based on php-ubuntu20.04 and one based on php-latest. 

The latest one (https://github.com/owncloud-docker/base/tree/master/latest) is not covered in your PR.

File: README.md (hunk 2 above)
Comment (voroyam) [reply README.md:324]: there is only `next` in the switcher menu

image

can you explain why you prefer latest?

Comment (xoxys) [reply README.md:324]: Because latest redirects always to the latest stable release while next container nor released features.

**Comment by xoxys** [comment]
> * declare statement missing (see https://github.com/owncloud-docker/base/blob/master/v20.04/overlay/etc/entrypoint.d/85-others.sh#L27-L28)

This one is still missing.

**Review by xoxys — APPROVED** [review]

**Pull request merged**